witnet.toml

```diff
@@ -10,44 +10,108 @@ public_addr = "0.0.0.0:21337"
+        "116.202.131.31:22339",
+        "116.202.131.32:22339",
+        "116.202.164.176:22339",
+        "116.202.35.247:22339",
+        "134.122.116.156:22339",
+        "134.209.84.237:22339",
+        "135.181.0.31:22339",
+        "135.181.194.161:21337",
+        "135.181.60.149:22345",
+        "136.243.135.114:22339",
+        "136.243.93.142:22352",
+        "136.243.94.171:22346",
+        "136.243.95.38:22342",
+        "136.243.95.38:22348",
+        "138.68.108.254:22339",
+        "142.93.42.49:22339",
+        "144.76.18.98:21337",
+        "144.91.113.168:21337",
+        "148.251.128.18:22348",
+        "148.251.128.19:22342",
+        "148.251.128.26:22341",
+        "157.245.171.146:21337",
+        "159.69.139.239:22339",
+        "159.69.56.28:22339",
+        "159.69.72.123:22339",
+        "159.69.74.96:22339",
+        "159.89.5.213:22339",
+        "159.89.7.193:22339",
+        "161.35.167.68:22339",
+        "161.35.235.109:22339",
+        "163.172.131.197:21337",
+        "165.232.106.11:22339",
+        "165.232.108.246:22339",
+        "165.232.32.106:22339",
+        "167.172.29.131:22339",
+        "167.172.39.205:22339",
+        "167.71.2.204:22339",
+        "167.71.5.253:22339",
+        "167.99.154.252:22339",
+        "167.99.200.189:22339",
+        "167.99.243.125:22339",
+        "167.99.246.148:22339",
+        "167.99.248.158:22339",
+        "167.99.249.236:22339",
+        "173.212.241.42:21337",
+        "173.249.40.145:25337",
+        "176.9.29.25:22342",
+        "178.62.221.245:22339",
+        "188.166.109.122:22339",
+        "188.166.59.159:22339",
+        "188.166.71.96:22339",
+        "188.40.103.83:21537",
+        "188.40.103.83:21637",
+        "188.40.103.83:22437",
+        "188.40.103.83:22737",
+        "188.40.131.24:22339",
+        "188.40.94.105:22339",
+        "192.241.148.38:22339",
+        "195.201.167.113:22339",
+        "195.201.173.77:22339",
+        "195.201.181.221:22339",
+        "195.201.181.245:22339",
+        "195.201.240.189:22339",
+        "206.189.63.220:22339",
+        "207.154.214.111:22339",
+        "207.154.238.90:22339",
+        "207.154.253.202:22339",
+        "207.154.254.153:22339",
+        "213.239.234.132:22348",
+        "46.4.104.114:22339",
+        "49.12.2.247:21337",
+        "5.189.172.149:24337",
+        "64.227.69.233:22339",
+        "64.227.74.14:22339",
+        "68.183.202.131:22339",
+        "78.46.86.104:22342",
+        "78.47.209.194:22339",
+        "81.30.157.7:22339",
+        "81.30.157.7:22345",
+        "81.30.157.7:22346",
+        "81.30.157.7:22348",
+        "81.30.157.7:22350",
+        "81.30.157.7:22351",
+        "81.30.157.7:22352",
+        "81.30.157.7:22353",
+        "85.114.132.66:22339",
+        "85.114.132.66:22348",
+        "85.114.132.66:22351",
+        "85.114.132.66:22352",
+        "94.130.165.149:22339",
+        "94.130.30.59:22339",
+        "95.217.181.71:21337",
```

Comment:
This list of peers is ordered lexicographically, not numerically

Reply:
Format police :police_officer: :rofl: